### DISCUSSION TITLE
DeepSNR integration

### DISCUSSION THREAD

**Discussion by amasu** [jarmoruuth/AutoIntegrate#37] [Ideas]
Hello,

First thank you very much for your work! 
I just started using AutoIntegrate and I can see you put a lot of love an dedication into this project :) 

Just an suggestion: since I saw you added the option of using Starnet for stars reduction, maybe it worth integrating DEEPSNR, from the same author, to the supported noise reduction tools?
Here is the home website: https://www.deepsnrastro.com 

Again, thanks a lot for your work :) 
Cheers!

**Comment by jarmoruuth** [comment]
Hi,

Thanks for the suggestion! No idea why I have not checked it before. I am on a summer vacation so depending on the weather I will take a look at it sooner or later :)

Jarmo

**Reply by amasu** [reply]
Sure take your time there is no hurry :) enjoy your holidays !

**Comment by jarmoruuth** [comment]
Hi,

I now have a test version with DeepSNR support available in my test repository: https://ruuth.xyz/test/autointegrate/

Jarmo

**Reply by amasu** [reply]
Hi,
Thank you!
I'll test it probably this weekend and drop a feedback if any :)

**Reply by amasu** [reply]
Hello,

I have just completed a couple of runs with Autointegrate (using DSNR instead of graXpert for denoising). It works like a charm so far :) nothing to report. Thank you again!

**Reply by jarmoruuth** [reply]
Great! Thanks for the info.

Jarmo
